CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+39.0.1
+------
+- Bump golang.org/x/net from v0.17.0 -> v0.23.0
+- Bump protobuf (protoc) from 21.5 -> 26.1
```

Comment:
Don't we generate and commit the protobuf stuff?  Or is it generated on compile?

Reply:
I thought we did too, but looks like some of the tooling has rotted or I am missing steps.

Reply:
Updated, just changed the comment.